lib/erb_lint/linters/rubocop.rb

```diff
@@ -63,6 +63,15 @@ def autocorrect(processed_source, offense)
         end
       end
 
+      def checksum
```

Comment:
Let's call this something more intention-revealing. Rails calls something similar `cache_key`, that might work.